src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputSkipIfLastAndSharedWithCase.java

```diff
@@ -0,0 +1,113 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputSkipIfLastAndSharedWithCase
+{
+    void method(int i) {
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+                break;
+            case 2:
+                break;
+        }
+
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+            case 2:
+                break;
+            case 3:
+                break;
+        }
+
+        switch (i) {
+            default: // No violation with the new option is expected
```

Comment:
@sagar-shah94 If we allow `default` anywhere, then what is the purpose of this check? Won't it be checking nothing then as it can produce violations?
Check is specifically for people who want `default` to be last in some form.